plugins/preflight.yaml

```diff
@@ -33,7 +33,31 @@ spec:
     sha256: "5d96a7b9abb3699fc495b8739519e7b608c1d42f8b22b1e6581b889de7376872"
     files:
     - from: "/preflight.exe"
-      to: "kubectl-preflight.exe"
-    bin: "kubectl-preflight.exe"
-  shortDescription: Executes preflight tests and application conformance verifications on a cluster before installing an application
+      to: "."
+    bin: "preflight.exe"
+  shortDescription: Executes application preflight tests in a cluster
   homepage: https://github.com/replicaetdhq/troubleshoot
+  caveats: |
+    Usage:
+      $ kubectl preflight <URL>
+
+    Flags:
+      -h, --help                          help for preflight
+          --interactive                   interactive preflights (default true)
```

Comment:
I don't think it makes much sense to repeat the output of `preflight -h` here. Mentioning the help flag is perfect, but the other flags are too much, in my opinion.